src/hosting/proxy.ts

```diff
@@ -68,6 +68,11 @@ export function proxyRequestHandler(url: string, rewriteIdentifier: string): Req
       Cookie: sessionCookie || "",
     });
     for (const key of Object.keys(req.headers)) {
+      // Skip particular header keys:
+      // - using x-forwarded-host, don't need to keep `host` in the headers.
+      if (["host"].includes(key)) {
```

Comment:
Why `["host"].includes(key)` and not `key === "host"`?  Was this maybe a longer list at first?

Reply:
Then maybe do two lines:
```ts
const headersToDelete = ["host"];

if (headersToDelete.includes(key)) {
```